docs/Developers/Executing_Tests.rst

```diff
@@ -7,7 +7,7 @@ Executing our Tests
 -------------------
 
 coala has a big test suite. It is meant to work on every platform on
-every PC. If you just execute our tests you are doing us a favor.
+every PC. If you just execute our tests you are doing us a favour.
```

Comment:
`favour` is the British English way of spelling the word - `favor` is equally as acceptable, so I don't believe this change is necessary.